test/unit/lodash.spec.js

```diff
@@ -0,0 +1,1080 @@
+'use strict';
+
+var chai = require('chai');
+chai.use(require('sinon-chai'));
+var expect = chai.expect;
+var chance = require('chance').Chance();
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var generator = require('./Generator');
+var sinon = require('sinon');
+
+describe('Lodash Collection training', function () {
+
+    describe('countBy', function () {
+        describe('countBy1 function', function () {
+            var params = datasets.countBy();
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Array;
+                var element2 = params[1] instanceof Function;
```

Comment:
In documentation first parametr could be Array|Object, and second parametr Array|Function|Object|string